tests/system/test_system.py

```diff
@@ -51,10 +51,12 @@
 from tests._fixtures import DDL_STATEMENTS
 from tests._fixtures import EMULATOR_DDL_STATEMENTS
 from tests._helpers import OpenTelemetryBase, HAS_OPENTELEMETRY_INSTALLED
+from google.cloud.spanner_v1 import RequestOptions
 
 
 CREATE_INSTANCE = os.getenv("GOOGLE_CLOUD_TESTS_CREATE_SPANNER_INSTANCE") is not None
 USE_EMULATOR = os.getenv("SPANNER_EMULATOR_HOST") is not None
+STAGING_API_ENDPOINT = os.getenv("SPANNER_STAGING_HOST_API_ENDPOINT") is not None
```

Comment:
nit: this could be any endpoint potentially right? Something like `SPANNER_API_ENDPOINT` might be more general.

Reply:
Thanks will use that instead.